<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedPref</class>
 <widget class="QWidget" name="AdvancedPref">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>476</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="kcfg_diskPrealloc">
     <property name="toolTip">
      <string>Whether or not diskspace should be reserved before starting to download a torrent.</string>
     </property>
     <property name="title">
      <string>Reserve disk space before starting a torrent</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="kcfg_fullDiskPrealloc">
        <property name="toolTip">
         <string>Instead of doing a quick reservation, do a full reservation. This is slower than the quick way, but avoids fragmentation on the disk.</string>
        </property>
        <property name="text">
         <string>Fully reserve disk space (avoids fragmentation)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Performance</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>GUI update interval:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="kcfg_guiUpdateInterval">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;p&gt;The interval in milliseconds between GUI updates.&lt;/p&gt;
&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note: &lt;/span&gt;Increasing this will decrease CPU usage.&lt;/p&gt;</string>
          </property>
          <property name="suffix">
           <string> ms</string>
          </property>
          <property name="minimum">
           <number>500</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Network sleep interval:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="kcfg_cpuUsage">
          <property name="toolTip">
           <string>&lt;p&gt;Amount of time the network threads will sleep when they are speed limited. This has absolutely no effect when there are no speed limits.&lt;/p&gt;
&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note:&lt;/span&gt; The lower this is the more CPU is used. Setting it high can lead to lower speeds in high bandwidth situations. &lt;/p&gt;
&lt;p&gt;For example on an 100 Mbit LAN, if you set a limit of 3000 KiB/s, you might not actually reach this speed when this value is too high. Without a limit you can easily get above 3000 KiB/s on a LAN.&lt;/p&gt;</string>
          </property>
          <property name="suffix">
           <string> ms</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>250</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>111</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Magnets Queue</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="GridLayout_1">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Number of downloading slots:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="kcfg_numMagnetDownloadingSlots">
          <property name="toolTip">
           <string>Maximum number of concurrent downloading magnets at the same time.</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="kcfg_requeueMagnets">
          <property name="toolTip">
           <string>Whether or not the magnets that are not downloaded after a maximum period of time must be pushed back at the end of the queue.</string>
          </property>
          <property name="text">
           <string>Requeue magnets after:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="kcfg_requeueMagnetsTime">
          <property name="toolTip">
           <string>Maximum time that a magnet can occupy a downloading slot before it will be pushed back in the queue.</string>
          </property>
          <property name="suffix">
           <string> min</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Preview size for audio files:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="kcfg_previewSizeAudio">
            <property name="toolTip">
             <string>&lt;p&gt;The amount of data at the beginning and end of an audio file, which will be prioritized for previewing.&lt;/p&gt;</string>
            </property>
            <property name="suffix">
             <string> KiB</string>
            </property>
            <property name="minimum">
             <number>16</number>
            </property>
            <property name="maximum">
             <number>100000000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Preview size for video files:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="kcfg_previewSizeVideo">
            <property name="toolTip">
             <string>&lt;p&gt;The amount of data at the beginning and end of a video file, which will be prioritized for previewing.&lt;/p&gt;</string>
            </property>
            <property name="suffix">
             <string> KiB</string>
            </property>
            <property name="minimum">
             <number>16</number>
            </property>
            <property name="maximum">
             <number>100000000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>111</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_useQHttpAnnounce">
        <property name="toolTip">
         <string>Disables the use of KDE's KIO library for tracker announces.

This is experimental and should only be used if you are getting a lot of connection to host broken errors.</string>
        </property>
        <property name="text">
         <string>Do not use KIO for tracker announces</string>
        </property>
        <property name="visible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_lookUpHostnameOfPeers">
        <property name="toolTip">
         <string>Resolve the hostname of each peer. This will result in the hostname of a peer being displayed instead of the IP address. 

It can be disabled if you do not like the additional network traffic it generates</string>
        </property>
        <property name="text">
         <string>Resolve hostnames of peers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>575</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
